cli/server.go

```diff
@@ -505,23 +505,21 @@ func Server(vip *viper.Viper, newAPI func(context.Context, *coderd.Options) (*co
 				defer serveHandler(ctx, logger, nil, cfg.Pprof.Address.Value, "pprof")()
 			}
 			if cfg.Prometheus.Enable.Value {
-				options.PrometheusRegistry = prometheus.NewRegistry()
-				closeUsersFunc, err := prometheusmetrics.ActiveUsers(ctx, options.PrometheusRegistry, options.Database, 0)
+				options.PrometheusRegisterer = prometheus.DefaultRegisterer
```

Comment:
This technically breaks tests, because it adds global state. How easy is it to attach the default metrics?